lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -560,6 +562,27 @@ def install_a_puppet_msi_on(host, opts)
           end
         end
 
+        # Installs Puppet and dependencies from FreeBSD ports
+        #
+        # @param [Host] host The host to install packages on
+        # @param [Hash{Symbol=>String}] opts An options hash
+        # @option opts [String] :version The version of Puppet to install (shows warning)
+        #
+        # @return nil
+        # @api private
+        def install_puppet_from_freebsd_ports_on( host, opts )
```

Comment:
No, as this is an _on method that doesn't work correctly with an array of hosts.